bin/inasafe.py

```diff
@@ -32,11 +33,15 @@
     QgsMapLayerRegistry,
     QgsCoordinateReferenceSystem)
 
+
+from safe.test.utilities import get_qgis_app
+# make sure this line executes first
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
```

Comment:
I didn't use the variable specifically, but get_qgis_app needs to be called first to initialize QPaint and other Qt related object for the composer to works.
Any solution besides this is highly appreciated.